src/common/utils/Utils.ts

```diff
@@ -51,6 +51,17 @@ export function getArraySize(array: string[]): number {
   return 0;
 }
 
+export function traverseObjectRecursively(obj: any, callback: (visitedObject: any) => void): void {
```

Comment:
@vblagoje See [ErrorModalCtrl](https://github.com/infinispan/infinispan-management-console/blob/master/src/common/dialogs/ErrorModalCtrl.ts#L15-L18) for an example. The parameter `value` in your callback is then equivalent to `obj[k]` in your function. 

Furthermore, we could improve the `traverse` function so that the callback parameter is more rigidly controlled like in your implementation. So instead of `callback: Function` we have:

```Typescript
export function traverse(obj: any, callback: (obj: any, value: any, trail: any[]) => void, trail?: any[]): void {
// content
}
```